SpatialGDK/Source/SpatialGDKEditor/Private/SpatialTestSettings.cpp

```diff
@@ -21,7 +21,8 @@ const FString FSpatialTestSettings::OverrideSettingsFileDirectoryName = TEXT("Ma
 const FString FSpatialTestSettings::OverrideSettingsFilePrefix = TEXT("TestOverrides");
 const FString FSpatialTestSettings::OverrideSettingsBaseFilename =
 	FPaths::ConvertRelativePathToFull(FPaths::ProjectConfigDir() / OverrideSettingsFileDirectoryName);
-const FString FSpatialTestSettings::BaseOverridesFilename = OverrideSettingsBaseFilename + TEXT("Base") + OverrideSettingsFileExtension;
+const FString FSpatialTestSettings::BaseOverridesFilename =
+	OverrideSettingsBaseFilename + TEXT("/") + OverrideSettingsFilePrefix + TEXT("Base") + OverrideSettingsFileExtension;
```

Comment:
```suggestion
	OverrideSettingsBaseFilename / OverrideSettingsFilePrefix + TEXT("Base") + OverrideSettingsFileExtension;
```
Pretty sure this should work, but up to you if you want to include it

Reply:
Yeah I was looking at that, but I am not keen to change code in flight.